cli/src/config/templates.toml

```diff
@@ -430,4 +430,4 @@ git_format_patch_email_headers = '''
 
 # Gerrit change id is 40 chars, jj change id is 32, so we need some padding.
 # `6a6a6964` is the hexadecimal of `jjid` in ASCII, just to not pad with zeros.
-"format_gerrit_change_id_trailer(commit)" = '"Change-Id: I6a6a6964" ++ commit.change_id().normal_hex() ++ "\n"'
+"format_gerrit_change_id_trailer(commit)" = 'if(!commit.description().contains("Change-Id: I"), "Change-Id: I6a6a6964" ++ commit.change_id().normal_hex() ++ "\n")'
```

Comment:
nit: better to test this by caller? it's odd that `format_*()` function behaves differently depending on the old description.

Reply:
By the caller, do you mean the user-configured template or the internal rust function? Or something else?

I get what you're saying, but since this trailer is based on "random" text (not something consistent like the user's name), it seems like this is a relatively unique case when checking for repetition.

Reply:
> By the caller, do you mean the user-configured template or the internal rust function? 

The user-configured template (and the documentation example.)

I think glehmann's idea is better, btw.
https://github.com/jj-vcs/jj/pull/6402#pullrequestreview-2791080674
> We could implement the check in the rust code, with a configurable list of trailer to keep unique?